### ISSUE TITLE
STAR Panel Request 8

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [iantaylor-NOAA/Lingcod_2021#145]
8. A) Remove all fixed gear age composition data. B) Remove fixed gear age composition data for the years 1999-2011. C) Do A and B with and without offsets for sex-based selectivity for all fleets. Requests are for North only model. Provide the model comparisons slides and table(s) with likelihoods and key parameter outputs, selectivity outputs, etc. The STAT will include the updated results from Day 2, Request 5.

Rationale:
These model runs are requested to better understand whether management changes or unbalanced periods of age sampling data in the associated fisheries are contributing disproportionally to tension in the model. There is an observed disconnect between the fixed gear length and CAAL in the 2000s that are more likely to reflect fleet behavior.

**Comment by iantaylor-NOAA** [comment]
Models set up using scripts and functions changed in commit 3fdd6ab. 
These models are running in sequence but I'm not going to wait for them to finish. I will upload results in the morning.

**Comment by iantaylor-NOAA** [comment]
I just committed figures and tables for this request as well as an updated `presentations/presentation_sandbox.Rmd` which includes those results. I think this fulfills the request. There's more that could be added but wanted to get this minimum done first.

I'll fix the table headers and maybe add some additional info related to this later.

Model files are uploaded to google drive: https://drive.google.com/drive/folders/1xkIZznTkVXHjhjWJOYJrMQO93qfD-O3V.

Here's a sample figure
![compare2_spawnbio_uncertainty](https://user-images.githubusercontent.com/4992918/125634702-af391431-2152-492d-b225-2ccc30965b5e.png)

**Comment by iantaylor-NOAA** [comment]
The "surgical removal" of 1999-2011 ages from the FG fleet had very little impact, suggesting that the non-representative ages were being adequately dealt with by representing the ages and conditioned on length. Removing all the FG ages had a bigger impact but we won't learn much from profiling this model as the profile for the old north base showed that ages from trawl and Rec WA were the most influential on M, not fixed-gear as shown in the fig below from `models/2021.n.022.001_new_INIT_profile_NatM_uniform_Fem_GP_1`. The models with sex-specific selectivity all look similar and we've profiled something very similar to those already.
![piner_panel_NatM_uniform_Fem_GP_1](https://user-images.githubusercontent.com/4992918/125635813-d854511a-35f0-4916-b287-ae8a8db4dd89.png)

Two useful directions that I see for pursuing are 
1. **removing all early fishery-dependent ages**
  perhaps everything prior to 1990 and then everything prior to 2000. If growth were time-varying that would reduce conflict between ages and lengths in the model. The difference between the light blue and green lines in my comment above show that it's either the early FG ages or the late ones which make the biggest difference, and I doubt it's the late ones.
2. **Exploring other functional forms for sex-specific selectivity**
   This might produce more reasonable results than what's present in the sensitivities explored so far.

I've made a start on both of these and will try to get something put together in time for the meeting.
